plugins/hazelcast37/src/main/java/org/radargun/service/Hazelcast37ClientService.java

```diff
@@ -0,0 +1,57 @@
+package org.radargun.service;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.MemberAttributeEvent;
+import com.hazelcast.core.MembershipEvent;
+import com.hazelcast.core.MembershipListener;
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.ProvidesTrait;
+
+/**
+ * Hazelcast client service
+ *
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+
+@Service(doc = "Hazelcast client")
+public class Hazelcast37ClientService extends Hazelcast36Service {
```

Comment:
I guess I don't see why Hazelcast shouldn't have multiple services, similar to the Infinispan plugin. Especially since I suspect different groups will use the different services in the benchmarks as well. Am I missing something?